### DISCUSSION TITLE
Calling GET /api/v2/token results in already expired token

### DISCUSSION THREAD

**Discussion by KMConnectTeam** [drakkan/sftpgo#1739] [Q&A]
Hello the title says it all, but i want to provide additional information about the setup.

Setup is as follows:
- SFTPGo v2.6 in a Docker container
- Postgres 16.3 as persistence 

When i call GET /api/v2/token, then i get an already expired token. More detailed: i live in Germany its CEST 15:34 but the token has expire date of 2024-08-30T13:54:58Z

I checked Docker container time, time of Postgresql in postgresql.conf. And no matter where i do changes the result is always the same.
I think the token is created to be valid for 20 minutes. For some reason it uses GMT (which is 2 hours behind). Then it can be calculated as follows: CEST 15:34 - 2 hours + 20 min = 13:54

Has anybody an idea ? Or am i the only one who faced this problem ?

**Comment by drakkan** [comment]
`2024-08-30T13:54:58Z` means UTC time, this token is not expired

**Comment by KMConnectTeam** [comment]
thanks Sir !
